src/cephadm/cephadm

```diff
@@ -588,7 +782,7 @@ class FileLock(object):
         # os.open() function.
         # This file lock is only NOT None, if the object currently holds the
         # lock.
-        self._lock_file_fd = None
+        self._lock_file_fd = -1 # type: int
```

Comment:
Does the changes to `_lock_file_fd` deserve it's own PR?

If not, what's the reason behind this change?

Reply:
mypy detected the problem.  ( if we use mypy to define types explicitly, it seems sensible to use the right type values when you are using a variable). In this case and can be seen in the documentation a file descriptor must be a small integer number. 
I thought it was ok to solve in this way., also we can tweak mypy annotations in order to solve it ...and allow to merge this pr ( it was failing before)